Sources/config.cpp

```diff
@@ -212,16 +212,20 @@ void from_json(nlohmann::json const& j, homescreenConfig& o) {
 }
 
 void to_json(nlohmann::json& j, Settings const& o) {
-  j = nlohmann::json{ { "ftp", nlohmann::json(o.ftp) },
+  j = nlohmann::json{ { "active_theme_directory", o.activeThemeDirectory },
+                      { "ftp", nlohmann::json(o.ftp) },
                       { "mount", nlohmann::json(o.mount) },
 #ifdef NXDK
                       { "network", nlohmann::json(o.net) },
 #endif
                       { "logging", nlohmann::json(o.logging) },
-                      { "homescreenConfig", nlohmann::json(o.homescreen) } };
+                      { "homescreen", nlohmann::json(o.homescreen) } };
```

Comment:
Nit: specific reason why this was changed? Listing it as config may make it easier for outside devs to understand.

Reply:
It's been a long time, but looking a few lines down at line 246 it's specifically looking for "homescreen" when reading the config file, so I think saving the config would always cause the default values to be used. 

It also strikes me as more consistent to use "homescreen" in the same way that "logging", etc... is used rather than "loggingConfig".

Reply:
Sun's God